mods/cnc/maps/gdi04c/gdi04c.lua

```diff
@@ -88,7 +88,7 @@ WorldLoaded = function()
 	end)
 
 	nodObjective = nod.AddPrimaryObjective("Destroy all GDI troops.")
-	gdiObjective1 = player.AddPrimaryObjective("Defend the town of Bialystok.")
+	gdiObjective1 = player.AddPrimaryObjective("Defend the town of Białystok.")
```

Comment:
Does this work? https://github.com/OpenRA/OpenRA/pull/6699#issuecomment-58279705